public/404.html

```diff
@@ -0,0 +1,152 @@
+<!--
```

Comment:
I don't think we need such an elaborate 404. it calls into question if it was copy/pasted etc. I don't think anyone here will be able to maintain it. Let's use a simple (no css) or default please?